latch_cli/services/workspace.py

```diff
@@ -7,24 +7,27 @@
 from latch_cli.utils import current_workspace, retrieve_or_login
 
 
-def workspace():
-    """Opens a terminal user interface in which a user can select the workspace
-    the want to switch to.
-
-    Like `get_executions`, this function should only be called from the CLI.
-    """
-
+def get_workspaces():
```

Comment:
```suggestion
def _get_workspaces():
```

so that it doesn't show up in API docs